cmd/netgoal/generate.go

```diff
@@ -61,6 +67,12 @@ func init() {
 	generateCmd.Flags().StringVarP(&nodeTemplatePath, "node-template", "", "", "json for one node")
 	generateCmd.Flags().StringVarP(&nonParticipatingNodeTemplatePath, "non-participating-node-template", "", "", "json for non participating node")
 	generateCmd.Flags().StringVarP(&relayTemplatePath, "relay-template", "", "", "json for a relay node")
+	generateCmd.Flags().StringVarP(&sourceWallet, "wallet-name", "", "", "Source wallet name")
+	generateCmd.Flags().Uint64VarP(&rounds, "rounds", "", 13, "Number of rounds")
+	generateCmd.Flags().Uint64VarP(&roundTrxCount, "ntxns", "", 17, "Transaction count")
+	generateCmd.Flags().Uint64VarP(&accountsCount, "naccounts", "", 31, "Account count")
+	generateCmd.Flags().Uint64VarP(&assetsCount, "nassets", "", 5, "Asset count")
+	generateCmd.Flags().Uint64VarP(&applicationCount, "napps", "", 7, "Application Count")
```

Comment:
why these default values?

Reply:
Wanted to create a small db file as default

Reply:
Random values for creating a small db file as default